src/patternfly/components/Check/examples/check-example.hbs

```diff
@@ -1,20 +1,20 @@
 {{#> check}}
-  {{#> check-input check-input--attribute='type="checkbox" id="check1" name="examplecheck1"'}}{{/check-input}}
+  {{#> check-input check-input--attribute='id="check1" name="exampleChecks"'}}{{/check-input}}
```

Comment:
should we keep the unique names for the checkboxes? I don't you commonly want checkboxes to share names do you?